test/Debugger/JsDiagGetScripts.js.dbg.baseline

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "sources": [
+      {
+        "scriptId": 3,
+        "fileName": "dummyfilename.js",
+        "lineCount": 1,

```

Comment:
Script type is only for dynamic script which doesn't have a filename. For such scripts filename is replaced with scriptType